java/src/jmri/jmrix/AbstractProgrammerFacade.java

```diff
@@ -13,6 +13,7 @@
  * facade classes.
  *
  * @author Bob Jacobsen Copyright (C) 2013
+ * @author Andrew Crosland Copyright (C) 2021
```

Comment:
It would be good to remove this comment now when the new `readCV` method is removed from this class.